python/lsst/obs/hsc/transformRegistry.py

```diff
@@ -1,9 +1,12 @@
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["transformRegistry"]
```

Comment:
Squash this part with the previous commit.  (But I don't think we should be exporting `transformRegistry`.)